### PR TITLE
feat(stonedb): update the copyright date time (#321)

### LINKED ISSUE #321: feature: need to update the copyright date time
**Is your feature request related to a problem? Please describe.**

now，the copyright date time is 2000~2022，that is not correct

**Describe the solution you'd like**

need to update the copyright date time

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by DandreChen**
## Summary about this PR

Issue Number: close #321 

## Tests Check List

- [ ] Unit test
- [x] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

## Changelog

- [ ] New Feature
- [ ] Bug Fix
- [x] Improvement
- [ ] Performance Improvement
- [ ] Build/Testing/CI/CD
- [ ] Documentation
- [ ] Not for changelog (changelog entry is not required)

## Documentation

- [x] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features

### CODE REVIEW

File: client/base/help_options.cc [hunk 1]
```diff
@@ -88,14 +88,14 @@ void Mysql::Tools::Base::Options::Help_options::print_usage()
 
   if (first_year_str == COPYRIGHT_NOTICE_CURRENT_YEAR)
   {
-    copyright= ORACLE_WELCOME_COPYRIGHT_NOTICE(COPYRIGHT_NOTICE_CURRENT_YEAR);
```
Comment (hustjieke) [inline client/base/help_options.cc:91]: Why 2021?
MySQL for 5.7: https://github.com/mysql/mysql-server/blob/5.7/client/base/help_options.cc#L98

Comment (DandreChen) [reply client/base/help_options.cc:91]: because stoneatom was founded in 2021，so,plan to start from 2021

Comment (hustjieke) [reply client/base/help_options.cc:91]: make sense.

File: client/base/help_options.cc [hunk 2]
```diff
@@ -88,14 +88,14 @@ void Mysql::Tools::Base::Options::Help_options::print_usage()
...
+    copyright= ORACLE_WELCOME_COPYRIGHT_NOTICE("2021");
   }
   else
   {
 #define FIRST_YEAR_CONSTANT "$first_year$"
     string first_year_constant_str= FIRST_YEAR_CONSTANT;
 
-    copyright= ORACLE_WELCOME_COPYRIGHT_NOTICE(FIRST_YEAR_CONSTANT);
+    copyright= ORACLE_WELCOME_COPYRIGHT_NOTICE("2021");
```
Comment (hustjieke) [inline client/base/help_options.cc]: Do not use magic number here.
Change all the `2021` to a macro definition.

Comment (DandreChen) [reply client/base/help_options.cc]: got it.  I have changed it

**Review by hustjieke — APPROVED** [review]
LGTM

**Comment by hustjieke** [comment]
ping @RingsC @zhx8817

**Review by isredstar — APPROVED** [review]
good job

**Pull request merged**